Week3/homework/js-exercises/Project_CreditCardValidator.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+function validNumber(num) {
+    
+    const sumDigits = str => { //This function sums the digits of the credit card number in the string. Below I used an if statement to make sure that sum of the digits is greater than 16.
+        let sum = 0;
+        for (let char of str) {
+          sum += +char;
+        }
+        return sum
+      };
+
+      function allSameDigits(str) {
+        return /^(\d)\1+$/.test(str.replace(/[^\d]/g,'')); //I used a regex function to check whether all digits are same. Is that okay to use a regex (I kind of feel like cheating)? 
+    }
+
+    if (creditCard.toString().length > 16 || creditCard.toString().length < 16) { //This if statement makes sure that there are 16 digits in the credit card number.
```

Comment:
`creditCard.toString().length > 16 || creditCard.toString().length < 16` can be replaced by simply `creditCard.toString().length !== 16`, right? No need to do an OR operation.